src/index.ts

```diff
@@ -1,12 +1,11 @@
 import { Express, Request, Response } from 'express';
 import { User, UserInput, ValidationResult } from './sharedTypes'
-import { TypeUtils } from './utils/TypeUtils';
+import { isUser } from './utils/typeUtils';
 import { UserValidation } from './user/UserValidation';
 import { UserActions } from './user/UserActions';
 import { UserData } from './user/UserData';
-import { GeneralUtils } from './utils/GeneralUtils';
-
 
+const utils = require('./utils/generalUtils')
```

Comment:
Please avoid using require when it's not needed and try to use import as much as possible.